libass/ass_fontselect.c

```diff
@@ -166,6 +170,111 @@ static ASS_FontProviderFuncs ft_funcs = {
     .destroy_font      = destroy_font_ft,
 };
 
+#if defined(_WIN32) && !defined(__CYGWIN__)
+/*
+ * Output parameters are only valid if the call was sucessful, ignore them otherwise.
+ * \param path          input; path of the dir to open, may be UTF-8 or the codepage used by fopen
+ * \param file_context  output
+ * \param find_handle   output; needs to be closed
+ * \param out_length    output
+ * \return utf-16le version of path; needs to be freed
+ */
+static wchar_t *win32_open_dir(const char *path, WIN32_FIND_DATAW *file_context, HANDLE *find_handle, size_t *out_length)
+{
+    UINT cur_cp = CP_UTF8;
```

Comment:
Nit: imho it would be better to move this closerto its usage, eg after `size_t wlen;`